src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEModuleSymbol.cs

```diff
@@ -702,5 +708,43 @@ internal IEnumerable<NamedTypeSymbol> GetForwardedTypes()
         }
 
         public override ModuleMetadata GetMetadata() => _module.GetNonDisposableMetadata();
+
+        internal bool ShouldDecodeNullableAttributes(Symbol symbol)
+        {
+            Debug.Assert(!(symbol is null));
+            Debug.Assert(symbol.IsDefinition);
+            Debug.Assert((object)symbol.ContainingModule == this);
+
+            if (_lazyNullableMemberMetadata == NullableMemberMetadata.Unknown)
+            {
+                _lazyNullableMemberMetadata = GetAttributes().IndexOfAttribute(this, AttributeDescription.NullablePublicOnlyAttribute) >= 0 ?
+                    (_assemblySymbol.GetAttributes().IndexOfAttribute(this, AttributeDescription.InternalsVisibleToAttribute) >= 0 ?
```

Comment:
>GetAttributes() [](start = 37, length = 15)

Should use well-know attributes infrastructure instead (special methods on PEAssembly/PEModule, etc.) Also, it feels like this will not handle .NET modules with InternalsVistibleTo attributes in them. Please add targeted tests. It might be much simpler to encode this information in the NullablePublicOnlyAttribute rather than trying to recalculate it back. Both for the compiler and other consumers.